products/dialogflow/terraform.yaml

```diff
@@ -0,0 +1,50 @@
+# Copyright 2020 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  Agent: !ruby/object:Overrides::Terraform::ResourceOverride
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "dialogflow_agent_full"
+        primary_resource_id: "full_agent"
+        vars:
+          agent_name: "tf-test-full-agent"
+    id_format: "{{project}}"
+    import_format: ["{{project}}"]
```

Comment:
This doesn't look quite right - I'm assuming you should have `project/{project}/agent/{agent-id}` or something like that.

Reply:
No agent-id, there is only one agent per project.

Reply:
Interesting! Can you leave a comment here? For when I forget about this conversation.